uc-catalog-cloning/example.ipynb

```diff
@@ -0,0 +1,133 @@
+      "            Creating Table tbl1_new and transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "         Creating Schema db2 and transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "         Creating Schema db_empty and transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "         Creating Schema db_ext and transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "            Creating Table tblx and transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "         Creating Schema db_func and transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "            Creating Table tbl1 and transferring permissions, comments and tags ...\u001b[0m \u001b[43m\u001b[31mQuery on table eo000_ctg_ext_loc2.db_func.tbl1 with row filter or column mask not supported on assigned clusters.\u001b[0m\n",
+      "            Creating Table tbl2 and transferring permissions, comments and tags ...\u001b[0m \u001b[43m\u001b[31mQuery on table eo000_ctg_ext_loc2.db_func.tbl2 with row filter or column mask not supported on assigned clusters.\u001b[0m\n",
+      "         Creating Schema db_other_assets and transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "            Creating Table tbl1 and transferring permissions, comments and tags ...\u001b[0m \u001b[43m\u001b[31mUser does not have USE SCHEMA on Schema 'eo000_ctg_ext_loc2.db_other_assets'.\u001b[0m\n",
+      "            Creating Table tbl2 and transferring permissions, comments and tags ...\u001b[0m \u001b[43m\u001b[31mUser does not have USE SCHEMA on Schema 'eo000_ctg_ext_loc2.db_other_assets'.\u001b[0m\n",
+      "            Creating Table valid_groups and transferring permissions, comments and tags ...\u001b[0m \u001b[43m\u001b[31mUser does not have USE SCHEMA on Schema 'eo000_ctg_ext_loc2.db_other_assets'.\u001b[0m\n",
+      "         Schema default already exists. Only transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n",
+      "            Creating Table tbl1 and transferring permissions, comments and tags ...\u001b[0m \u001b[43m\u001b[31mUser does not have SELECT on Table 'eo000_ctg_ext_loc2.default.tbl1'.\u001b[0m\n",
+      "         Schema information_schema already exists. Only transferring permissions, comments and tags ...\u001b[0m \u001b[32mDONE!\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "#If you are cloning a Catalog with a managed external location\n",
+    "inputs = dict(\n",
+    "    source_catalog_external_location_name = 'eo000_ext_loc_ctg2',\n",
+    "    source_catalog_name = 'eo000_ctg_ext_loc2',\n",
+    "    target_catalog_external_location_pre_req = ['eo000_ext_loc_ctg6', 'field_demos_credential', 'abfss://eo000ext6@oneenvadls.dfs.core.windows.net/'],\n",
```

Comment:
are you 1000% sure it's okay to refer to internal storage account names?

Reply:
changed them to more generic names